egs/iam/s5/local/chain/compare_wer.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# this script is used for comparing decoding results between systems.
+# e.g. local/chain/compare_wer.sh exp/chain/cnn{1a,1b}
+
+# ./local/chain/compare_wer.sh exp/chain/cnn1a exp/chain_cleaned/cnn1b
+# System                    cnn1a     cnn1b
+#     18.83     18.61
+# Final train prob        -0.0262   -0.0366
+# Final valid prob        -0.1340   -0.1708
+# Final train prob (xent)   -0.6418   -0.6389
+# Final valid prob (xent)   -1.1350   -1.1561
+
+if [ $# == 0 ]; then
+  echo "Usage: $0: <dir1> [<dir2> ... ]"
+  echo "e.g.: $0 exp/chain/cnn{1a,1b}"
+  exit 1
+fi
+
+echo "# $0 $*"
+used_epochs=false
+
+echo -n "# System               "
+for x in $*; do   printf "% 10s" " $(basename $x)";   done
+echo
+
+for x in $*; do
```

Comment:
Add a `echo -n "# WER            "` before this for.
Also, please tune the number of spaces so that all numbers (are vertically aligned)